opendevin/lib/command_manager.py

```diff
@@ -81,7 +82,8 @@ def get_background_events(self) -> List[Event]:
             if exit_code is not None:
                 events.append(Event('output', {
                     'exit_code': exit_code,
-                    'output': 'Background command %d exited with code %d' % (idx, exit_code),
+                    # TODO: undefined name 'idx'
```

Comment:
left this one in
afraid it might create bugs later on, so going to keep original code in for reference.